### PR TITLE
issue #37 #33. Refactoring for languages completed.

### PR DESCRIPTION
**Pull request by ortizh**

### CODE REVIEW

**Comment by nim442** [comment]
You need to merge dev into your branch before pull request

**Comment by ortizh** [comment]
@nim442 Conflict has been resolved. Dev branch was merged to Languages branch. You may proceed to code review.

File: mRides-app/MainActivity.cs [hunk 1]
```diff
@@ -90,22 +90,22 @@ void LoginToFacebook(bool allowCancel)
                     
                     // If the user already exists, set the current user to it
                     // and go to map activity
-                    if (user != null)
-                    {
-                        User.currentUser = user;
-                        var mapActivity = new Intent(this, typeof(MapActivity));
-                        StartActivity(mapActivity);
-                    }
+                   // if (user != null)
```
Comment (nim442) [inline mRides-app/MainActivity.cs]: I believe you forgot to uncomment this. This will create problems with database

Comment (ortizh) [inline mRides-app/MainActivity.cs]: @nim442 issue was fixed.

Comment (nim442) [inline mRides-app/MainActivity.cs]: perfect gj

**Pull request merged**